qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -1,115 +1,401 @@
+    $("#ena-ontology").change(function ( event ) {
+      var value = $("#ena-ontology").val();
+      if( value == "Other" ) {
+        $("#user-ena-info").show();
+        $("#user-ena-info").trigger('change');
+      }
+      else {
+        $("#user-ena-info").hide();
+        $("#new-ena-info").hide();
+        update_investigation_type(value);
+      }
     });
+    // If the user-ena-info select changes, check if we need to show the input box
     $("#user-ena-info").change(function ( event ) {
-        if( $("#user-ontology").val() == "New Type" ) { $("#new-ena-info").show(); }
-        else { $("#new-ena-info").hide(); }
+        var value = $("#user-ontology").val();
+        if( value == "New Type" ) {
+          $("#new-ena-info").show();
+        }
+        else {
+          $("#new-ena-info").hide();
+          if( value !== "" ) {
+            update_investigation_type(value);
+          }
+        }
     });
   });
-</script>
\ No newline at end of file
+</script>
+
+<!-- Prep template title and buttons -->
+<div class="row">
+  <div class="col-md-12">
+    <h4>
+      {{name}} - ID {{prep_id}}
+      <a class="btn btn-default glyphicon glyphicon-download-alt" href="/download/{{download_prep}}" style="word-spacing: -10px;"> Prep info</a>
+      <a class="btn btn-default glyphicon glyphicon-download-alt" href="/download/{{download_qiime}}" style="word-spacing: -10px;"> Qiime map</a>
+      <a class="btn btn-danger glyphicon glyphicon-trash" style="word-spacing: -10px;" onclick="delete_prep_info({{prep_id}});"> Delete</a>
+    </h4>
+  </div>
+</div>
+
+<!-- #samples and #columns -->
+<div class="row">
+  <div class="col-md-12">
+    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.
+  </div>
+</div>
+
+<!-- Update prep template -->
+<div class="row">
+  <div class="col-md-12">
+    <b>Update information:</b>
+    <select id="file-selector">
+      <option value="">Choose file...</option>
+      {% for fp in files %}
+        <option value="{{fp}}">{{fp}}</option>
+      {% end %}
+    </select>
+    <div id="update-button-div" hidden>
+      <button class="btn btn-info btn-sm" onclick="update_prep_information();" hidden>Update</button>
+    </div>
+  </div>
+</div>
+
+<!-- Investigation type info -->
+<div class="row">
+  <!-- Ena ontology selector -->
+  <div class="col-md-4">
+    <b>Select Investigation Type:</b>
+    <select id="ena-ontology" name="ena-ontology" value="ena-ontology">
+      <option value=""></option>
+      {% for o in ontology['ENA'] %}
+        <option value="{{o}}">{{o}}</option>
+      {% end %}
+    </select>
+  </div>
+  <!-- user-defined selector -->
+  <div class="col-md-4" id="user-ena-info" hidden> User defined investigation type:
+    <select id="user-ontology" name="user-ontology" value="user-ontology">
+      <option value=""></option>
+      {% for o in ontology['User'] %}
+        <option value="{{o}}">{{o}}</option>
```

Comment:
I guess we can put some validation on the user input, so only alphanumeric values and the space can be added as a user-defined term for the ENA ontology. Does this sound good to you?